recipe/build.sh

```diff
@@ -10,3 +10,5 @@ fi
 make
 make check
 make install
+
+mv ${PREFIX}/lib/*/include/* ${PREFIX}/include
```

Comment:
Agreed. However, it didn't work as detailed in the description. See 83dca57b6b82a28ede4b0d6ce51b3a9b679d1594 that didn't work on my [fork](https://circleci.com/gh/shadowwalkersb/libffi-feedstock/3).